pkg/process/checks/process.go

```diff
@@ -69,6 +70,9 @@ type ProcessCheck struct {
 	// disallowList to hide processes
 	disallowList []*regexp.Regexp
 
+	// determine if zombies process will be collected
+	ignoreZombieProcesses bool
```

Comment:
Per @wiyu prior comment - should this be `skipZombieProcesses` to be more inline with the naming of the `skipProcess`, then it just makes it trivial to understand.